packages/tools/devtools/devtools-example/webpack.config.js

```diff
@@ -25,6 +27,7 @@ module.exports = (env) => {
 					{
 						test: /\.tsx?$/,
 						loader: require.resolve("ts-loader"),
+						exclude: /node_modules/,
```

Comment:
I believe this is the default behavior. Do we need this line?

Reply:
Tested it. Works without it.